src/lib/containers/table/SynapseTable.tsx

```diff
@@ -394,7 +395,7 @@ export default class SynapseTable extends React.Component<
               </div>
             )}
           {/* FRAGILE, CHANGE WITH CAUTION, see - https://sagebionetworks.jira.com/browse/PORTALS-1539 */}
```

Comment:
It looks like the "fragile" part of this is around the table being in a div (so the table is always in the DOM).